autolab/Autolab
# Enable Rubocop Linting within /spec directory

## Description


As we move towards increasing test coverage across the Autolab repository, This PR enables rubocop linting across the directory, and also fixes linting errors brought on by enabling rubocop

## How Has This Been Tested?



- Setup testing db
```
RAILS_ENV=test bundle exec rails autolab:setup_test_env
```
- Run tests, see that all tests still pass
```
bundle exec rails spec
```

## Checklist:
